sys/aarch64/genassym.cf

```diff
@@ -5,6 +5,8 @@ include <sys/proc.h>
 include <aarch64/mcontext.h>
 include <aarch64/armreg.h>
 
+define DAIF_I_MASKED DAIF_I_MASKED
```

Comment:
Please add `-Werror` to `ASFLAGS` in `build/flags.mk` and clean up warnings.

Reply:
done